--- module/plugins/hooks/XFileSharingPro.py.orig	2015-10-07 17:14:42.151741017 +0200
+++ module/plugins/hooks/XFileSharingPro.py	2015-10-07 17:15:05.535794011 +0200
@@ -11,7 +11,7 @@
     __version__ = "0.43"
     __status__  = "testing"
 
-    __config__ = [("activated"       , "bool", "Activated"                     , True ),
+    __config__ = [("activated"       , "bool", "Activated"                     , False),
                   ("use_hoster_list" , "bool", "Load listed hosters only"      , False),
                   ("use_crypter_list", "bool", "Load listed crypters only"     , False),
                   ("use_builtin_list", "bool", "Load built-in plugin list"     , True ),
