--- ccutil/helpers.h.orig	2016-01-29 21:01:24.597470800 +0100
+++ ccutil/helpers.h	2016-01-29 21:02:13.397470800 +0100
@@ -61,8 +61,8 @@
  private:
   // Steps the generator to the next value.
   void Iterate() {
-    seed_ *= 6364136223846793005;
-    seed_ += 1442695040888963407;
+    seed_ *= 6364136223846793005ULL;
+    seed_ += 1442695040888963407ULL;
   }
 
   // The current value of the seed.
